packages/x-components/tests/unit/base-modal.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { mount } from '@cypress/vue';
+import Vue from 'vue';
+import BaseModal from '../../src/components/modals/base-modal.vue';
+import { BaseXBus } from '../../src/plugins/x-bus';
+import { XPlugin } from '../../src/plugins/x-plugin';
+import { e2eAdapter } from '../../src/adapter/e2e-adapter';
+
+/**
+ * Renders an {@link BaseModal} component with the provided options.
+ *
+ * @param  options - Options to render {@link BaseModal} with.
+ * @returns Helper methods for the rendered {@link BaseModal}.
+ */
+function renderBaseModal({
+  clientHeaderSelector = '',
+  clientHeaderHeight = {},
+  template = `
+      <div class="wrapper">
+        <div class="header"></div>
+        <BaseModal :clientHeaderSelector="clientHeaderSelector" :open="open" />
```

Comment:
As you are not testing different values of the `open` property, shouldn't we just pass `true` here and remove it from the `props` section? We either test it or remove it, as you prefer.